BraintreeCore/src/main/java/com/braintreepayments/api/core/AnalyticsApi.kt

```diff
@@ -133,6 +135,8 @@ internal class AnalyticsApi(
         private const val FPTI_KEY_BUTTON_TYPE = "button_type"
         private const val FPTI_KEY_BUTTON_POSITION = "button_position"
         private const val FPTI_KEY_PAGE_TYPE = "page_type"
+        private const val FPTI_KEY_MERCHANT_ENABLED_APP_SWITCH = "merchant_enabled_app_switch"
+        private const val FPTI_KEY_PAYPAL_ATTEMPTED_APP_SWITCH = "attempted_app_switch"
```

Comment:
Are we holding on merge/release until these tags are added FPTI? Not sure if there were existing ones that could be used or we want these specific names

Reply:
Unfortunately, there weren't any tags that we could reuse. I think we can hold the release until the new tags are approved. I do already see them showing up in FPTI explorer.

Reply:
Both of these are now listed as approved in FPTI so we should be good to merge/release whenever. 🚀